Sources/FoundationExtensions/OperationQueue+Extensions.swift

```diff
@@ -22,6 +22,10 @@ extension OperationQueue {
         switch cacheStatus {
         case .firstCallbackAddedToList:
             self.addOperation(factory.create())
+
+            Logger.verbose(Strings.network.enqueing_operation(factory.operationType,
```

Comment:
Equivalent log to the one for `addedToExistingInFlightList`, useful for debugging.